x/ccv/provider/keeper/relay.go

```diff
@@ -404,60 +420,25 @@ func (k Keeper) HandleSlashPacket(ctx sdk.Context, chainID string, data ccv.Slas
 		return
 	}
 
-	// slash and jail validator according to their infraction type
-	// and using the provider chain parameters
-	var (
-		jailTime      time.Time
-		slashFraction sdk.Dec
-	)
-
 	infractionHeight, found := k.getMappedInfractionHeight(ctx, chainID, data.ValsetUpdateId)
 	if !found {
 		k.Logger(ctx).Error("infraction height not found. But was found during slash packet validation")
 		// drop packet
 		return
 	}
 
-	switch data.Infraction {
-	case stakingtypes.Downtime:
-		// set the downtime slash fraction and duration
-		// then append the validator address to the slash ack for its chain id
-		slashFraction = k.slashingKeeper.SlashFractionDowntime(ctx)
-		jailTime = ctx.BlockTime().Add(k.slashingKeeper.DowntimeJailDuration(ctx))
-		k.AppendSlashAck(ctx, chainID, providerConsAddr.String())
-	case stakingtypes.DoubleSign:
-		// set double-signing slash fraction and infinite jail duration
-		// then tombstone the validator
-		slashFraction = k.slashingKeeper.SlashFractionDoubleSign(ctx)
-		jailTime = evidencetypes.DoubleSignJailEndTime
-		k.slashingKeeper.Tombstone(ctx, providerConsAddr)
-	}
+	// Note: the SlashPacket is for downtime infraction, as SlashPackets
+	// for double-signing infractions are already dropped when received
 
-	// slash validator
-	k.stakingKeeper.Slash(
-		ctx,
-		providerConsAddr,
-		int64(infractionHeight),
-		data.Validator.Power,
-		slashFraction,
-		data.Infraction,
-	)
+	// append the validator address to the slash ack for its chain id
+	k.AppendSlashAck(ctx, chainID, providerConsAddr.String())
 
 	// jail validator
 	if !validator.IsJailed() {
 		k.stakingKeeper.Jail(ctx, providerConsAddr)
 		k.Logger(ctx).Info("validator jailed", "provider cons addr", providerConsAddr.String())
 	}
 
-	k.slashingKeeper.JailUntil(ctx, providerConsAddr, jailTime)
```

Comment:
@mpoke @sainoe I'm not sure we can remove this call to `JailUntil` altogether. Shouldn't the call just be moved into the if-statement where `Jail` is called? What happens if you jail a validator without setting `JailUntil`, and how does it interact with [this check](https://github.com/cosmos/cosmos-sdk/blob/9c145c827001222df2e3e1101010874aeac20997/x/slashing/keeper/unjail.go#L55)?

Reply:
Don't think it causes a problem. AFAIU, we want validators to be able to unjail directly after beeing jailed. So by keeping the `JailUntil` untouched, i.e. to its zero value  1970-01-01 00:00:00 UTC,  [this check](https://github.com/cosmos/cosmos-sdk/blob/9c145c827001222df2e3e1101010874aeac20997/x/slashing/keeper/unjail.go#L55) won't go through and the unjailing will work.

Reply:
Addressed in https://github.com/cosmos/interchain-security/pull/692/commits/69086f8b6b05f2b654d5d42d52cee59d41e7ab0b